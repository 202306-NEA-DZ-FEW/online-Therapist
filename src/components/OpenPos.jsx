import Link from "next/link";
import React from "react";

function OpenPos({ title, description, type, typeInfo }) {
    return (
        <div>
            <div className='block lg:grid lg:grid-cols-6 lg:gap-7'>
                <div className='col-span-5'>
                    <Link href='/contact'>
                        <p className='text-xl lg:text-2xl text-Teal hover:underline hover:cursor-pointer'>
                            {title}
                        </p>
                    </Link>
                    <p className='text-lg lg:text-xl text-Gray hidden lg:block'>
                        {description}
                    </p>
                </div>
                <div>
                    <p className='text-xl lg:text-2xl text-Teal hidden lg:block'>
                        {type}
                    </p>
                    <p className='text-lg lg:text-xl text-Gray'>{typeInfo}</p>
                </div>
            </div>
            <hr className='h-px mt-5 mb-2 bg-Gray border-0 dark:bg-gray-700' />
        </div>
    );
}

export default OpenPos;
