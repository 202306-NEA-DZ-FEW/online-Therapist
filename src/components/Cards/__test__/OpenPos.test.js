import renderer from "react-test-renderer";

import OpenPos from "../OpenPos";

it("renders correctly", () => {
    const tree = renderer.create(<OpenPos />).toJSON();
    expect(tree).toMatchSnapshot();
});
