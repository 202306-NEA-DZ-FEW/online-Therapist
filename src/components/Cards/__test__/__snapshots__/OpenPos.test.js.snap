// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="block lg:grid lg:grid-cols-6 lg:gap-7"
  >
    <div
      className="col-span-5"
    >
      <a
        href="/contact"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <p
          className="text-xl lg:text-2xl text-Teal hover:underline hover:cursor-pointer"
        />
      </a>
      <p
        className="text-lg lg:text-xl text-Gray hidden lg:block"
      />
    </div>
    <div>
      <p
        className="text-xl lg:text-2xl text-Teal hidden lg:block"
      />
      <p
        className="text-lg lg:text-xl text-Gray"
      />
    </div>
  </div>
  <hr
    className="h-px mt-5 mb-2 bg-Gray border-0 dark:bg-gray-700"
  />
</div>
`;
